packages/kolibri-tools/lib/cli.js

```diff
@@ -534,7 +535,7 @@ function _addPathOptions(cmd) {
       '-i, --ignore <patterns...>',
       'Ignore these comma separated patterns',
       list,
-      ignoreDefaults
+      langIgnoreDefaults.length ? langIgnoreDefaults : ignoreDefaults
```

Comment:
So, this is a way to use `langIgnoreDefaults` to override `ignoreDefaults`? And, in the context of this PR, it's saying, "ignore all things in node_modules other than kolibri common", because that's what 'ignore' is. Is that right?

Reply:
The reason this was needed is that because of the way that yarn handles its workspaces, the `packages` folder is aliased/linked via the `node_modules` folder to allow packages to be discovered by regular node module resolution mechanisms.

Reply:
great - thanks!